### DISCUSSION TITLE
V2 install script broken

### DISCUSSION THREAD

**Discussion by Vendo232** [gtxaspec/wz_mini_hacks#200] [General]
the script creates 2MB demo.bin which bricks the V2 cam. workaround is to download FW from Wyze web and put it in the v2_ro folder ( create the v2_ro folder inside V2_install )

![image](https://user-images.githubusercontent.com/44098230/180322194-f65c7a3e-640a-45a8-b5db-ff9c2580024e.png)

**Comment by Vendo232** [comment]
after the demo.bin install the V2 can not connect to local network. it says " I can not find network specificied... " any idea anyone what I`m doing wrong?

**Comment by gtxaspec** [comment]
.1002 firmware version? fresh wz mini install?

**Comment by Vendo232** [comment]
OK issue close, it was the SD CARD formatting software on PC/Windows

minitool partition wizard ( does not work ) 
guiformat.exe ( work )

both formatting sdcard to FAT32.

**Comment by baudneo** [comment]
Sorry I'm back again. I have v2 up and running but the RTSP streams are not working.

I followed the directions using compile_image.sh, the camera is running the firmware and `netstat -tulpn` shows that v4l2rtspserver is running on port 8554 BUT no RTSP streams can be grabbed.

```
    ~  ssh wyze2                                                                                                                             ✔  0.13   37.2G   04:53:35 PM   ⇣0 B/s ⇡0 B/s 10.0.1.9  
COLUMNS=213;LINES=50;export COLUMNS LINES;
[root@WCV2:~]# netstat -tulpn
Active Internet connections (only servers)
Proto Recv-Q Send-Q Local Address           Foreign Address         State       PID/Program name
tcp        0      0 0.0.0.0:4000            0.0.0.0:*               LISTEN      224/iCamera
tcp        0      0 0.0.0.0:8554            0.0.0.0:*               LISTEN      444/v4l2rtspserver
tcp        0      0 127.0.0.1:53            0.0.0.0:*               LISTEN      454/dnsmasq
tcp        0      0 0.0.0.0:22              0.0.0.0:*               LISTEN      99/dropbear
tcp        0      0 :::8554                 :::*                    LISTEN      444/v4l2rtspserver
tcp        0      0 :::22                   :::*                    LISTEN      99/dropbear
udp        0      0 127.0.0.1:53            0.0.0.0:*                           454/dnsmasq
udp        0      0 0.0.0.0:42829           0.0.0.0:*                           224/iCamera
udp        0      0 0.0.0.0:5555            0.0.0.0:*                           224/iCamera
udp        0      0 0.0.0.0:32761           0.0.0.0:*                           224/iCamera
```

The web server is enabled, and the image box just shows a broken image link. Zoneminder throws this error -> `Aug 07 16:54:08 ZoneMinder zmc_m9[32548]: ERR [zmc_m9] [Unable to open input rtsp://10.0.1.57:8554/video2_unicast due to: Server returned 404 Not Found]` . I have tried `video1_unicast`, `video2_unicast` and `unicast` endpoints.

Do I need to download a specific firmware version and create the dir structures for local FW file to be used to compile a new image? I saw in one of the discussion posts that a user said if the webserver doesnt show a RTSP stream in the UI that whatever hook used for RTSP was patched by wyze.

Attached is a screenshot of the webserver
![Screenshot_2022-08-07_17-09-31](https://user-images.githubusercontent.com/86508179/183314631-75d6ae43-095e-4313-8321-29c3ff02cbe4.png)

**Reply by gtxaspec** [reply]
can you run `gather_wz_logs.sh` and upload the logs for analysis?

**Reply by gtxaspec** [reply]
@baudneo i see this issue.  on the v2, you have to use `video6` and `video7` in the url.

**Reply by baudneo** [reply]
Ah crap I didn't catch that in the readme.md, thank you!
